$OpenBSD: patch-kernel_sig_info_cc,v 1.1 2017/06/03 14:25:16 espie Exp $
don't predeclare std before we have anything to put in there

Index: kernel/sig_info.cc
--- kernel/sig_info.cc.orig
+++ kernel/sig_info.cc
@@ -1,6 +1,5 @@
 #define KERNEL // Include internal kernel definitions
 
-using namespace std;
 #include <freehdl/kernel-error.hh>
 #include <freehdl/kernel-db.hh>
 #include <freehdl/kernel-sig-info.hh>
@@ -10,6 +9,7 @@ using namespace std;
 #include <freehdl/kernel-register.hh>
 #include <freehdl/kernel-signal-source-list-array.hh>
 #include <freehdl/kernel-map-list.hh>
+using namespace std;
 
 // signal_table is a hash tables (map) to associate signal names with
 // the corresponding sig_info_base pointers.
